test/tools/api.js

```diff
@@ -1,12 +1,14 @@
 /* eslint-disable prettier/prettier */
 'use strict';
 
+const { byStrings, sorted } = require("./utils");
```

Comment:
I have prettier ignored for this file so that the array stays one line per item, but now that I'm not editing it as frequently we can let it format, thoughts?
(We were only linting the src tho, added test!)

Reply:
I don't mind